ome/omero-figure
# Fix demo build

Updates the ```grunt demo``` task to work with the new build layout.

To test:
 - setup dev environment if not already done ```npm install```
 - ```grunt demo```
 - ```cd demo```
 - ```python -m SimpleHTTPServer```
 - Go to http://localhost:8000/ should load with no errors
 - Open file -> list of figures displayed
 - Pick a file -> should load correctly

cc @carandraug
